lib/interfaces/reddit.ts

```diff
@@ -0,0 +1,27 @@
+export class RedditPost {
```

Comment:
make this an interface

Reply:
alright

Reply:
it needs the constructor so it wouldn't really work

Reply:
it doesn't if you make it an interface and use an object

Reply:
i like classes :d

Reply:
i spent 20 minutes trying to figure it out and got frustrated so idk